mongodb-collection.js

```diff
@@ -21,9 +19,7 @@ class MongoDBCollection {
             return await this.collection.aggregate(pipeline).toArray()
         }
         catch (e){
-            console.debug("An exception occurred during an aggregate operation");
-            console.debug(e);
-            throw DATABASE_ERROR;
+            logAndThrow("An exception occurred during an aggregate operation", e);
```

Comment:
It is okay to keep try and catch clause. But, if you can minimize all of these, please do so.